cbits/glew.c

```diff
@@ -1,12 +1,16 @@
+#if defined(mingw32_HOST_OS)
 #include <GL/glew.h>
+#endif
 #include <stdio.h>
 #include "nanovg.h"
 #include "math.h"
 
 void initGlew() {
+	#if defined(mingw32_HOST_OS)
```

Comment:
Please put preprocessor directives at the beginning of the line rather than indenting them.